### PR TITLE
Allow to render a header with and without a GetFirefoxBanner for the blog

### PR DESCRIPTION
**Pull request by willdurand**
Fixes https://github.com/mozilla/addons-frontend/issues/10713

### CODE REVIEW

File: src/amo/components/Notice/styles.scss [hunk 1]
```diff
@@ -33,7 +33,6 @@
 }
 
 .Notice-dismisser {
-  align-items: top;
```
Comment (willdurand) [inline src/amo/components/Notice/styles.scss:36]: Both Chrome and Firefox said the value was invalid.

**Comment by willdurand** [comment]
@bobsilverberg wait, while trying out this patch in addons-blog, I noticed that the text says:

> To use these add-ons, you'll need to download FF

but... the blog index does not link to any add-ons. I believe this makes the whole feature invalid.

**Comment by bobsilverberg** [comment]
> @bobsilverberg wait, while trying out this patch in addons-blog, I noticed that the text says:
> 
> > To use these add-ons, you'll need to download FF
> 
> but... the blog index does not link to any add-ons. I believe this makes the whole feature invalid. See: [mozilla/addons-blog#228 (comment)](https://github.com/mozilla/addons-blog/issues/228#issuecomment-865817908)

I saw your other comment and I agree. I guess we wait to see what Jorge says. Regardless, we'll still need a way to hide the banner on either some, or all, blog pages, correct?

**Comment by willdurand** [comment]
> I saw your other comment and I agree. I guess we wait to see what Jorge says. Regardless, we'll still need a way to hide the banner on either some, or all, blog pages, correct?

Currently, the banner is never shown so we might not have to do extra work.

**Pull request closed without merging**